plugin-nabu/src/main/java/ca/on/oicr/gsi/shesmu/nabu/NabuPluginType.java

```diff
@@ -0,0 +1,26 @@
+package ca.on.oicr.gsi.shesmu.nabu;
+
+import ca.on.oicr.gsi.shesmu.plugin.Definer;
+import ca.on.oicr.gsi.shesmu.plugin.PluginFileType;
+import java.io.PrintStream;
+import java.lang.invoke.MethodHandles;
+import java.nio.file.Path;
+import org.kohsuke.MetaInfServices;
+
+@MetaInfServices
+public class NabuPluginType extends PluginFileType<NabuPlugin> {
+
+  public NabuPluginType() {
+    super(MethodHandles.lookup(), NabuPlugin.class, ".nabu", "nabu");
+  }
+
+  @Override
+  public NabuPlugin create(Path filePath, String instanceName, Definer<NabuPlugin> definer) {
+    return new NabuPlugin(filePath, instanceName, definer);
+  }
+
+  @Override
+  public void writeJavaScriptRenderer(PrintStream writer) {
+    // do nothing right now
```

Comment:
This method has an empty implementation in PluginFileType so this method can be removed entirely if you don't intend to implement this

Reply:
The plan is to eventually implement a CreateArchive action, which will implement this.